internal/pkg/agent/cmd/run.go

```diff
@@ -135,22 +142,33 @@ func run(streams *cli.IOStreams, override cfgOverrider) error {
 
 	statusCtrl := status.NewController(logger)
 
+	tracer, err := initTracer(agentName, release.Version(), cfg.Settings.MonitoringConfig)
+	if err != nil {
+		return err
+	}
+	if tracer != nil {
+		defer func() {
+			tracer.Flush(nil)
+			tracer.Close()
+		}()
+	}
```

Comment:
[Suggestion]
It'd be good to log if the APM is enabled or not, either here or on the `initTracer` function.